db/migrate/20230424093735_create_sync_dqt_induction_start_date_error.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateSyncDqtInductionStartDateError < ActiveRecord::Migration[6.1]
+  def change
+    create_table :sync_dqt_induction_start_date_errors, id: false do |t|
+      t.references :participant_profile, null: false, primary_key: true, foreign_key: true, type: :uuid, index: { name: "dqt_sync_participant_profile_id" }
+      t.text :error_message
+      t.timestamps
+    end
```

Comment:
I would prefer that a participant profile can have_many :sync_dqt_induction_start_date_errors: 
each induction record of the participant can have a different issue when amending their cohort (i.e target cohort not setup in our service, target school_cohort not setup by the school, target school_cohort with no default induction programme...). 

If we have each of these errors in their own record, then we can later on query the db for specific ones (give me all errors where school_cohort has not been setup by the school, for instance) and do a comms campaign to all the affected schools.

Your implementation joins all the errors of a participant in one huge string which would make it more difficult to query for the purposes above.

Reply:
@ltello it seems a bit overkilled as we want to save the errors from the latest run of the job.
I think the point is just to have a list of participants for which the job failed and that needs attention.
I am not sure we need an error record for each error message.

Reply:
The idea is to be able to query the table to tackle all the participants/schools with issue A, all the participants/schools with issue B, etc... and do a different action onto each group